mu/locale/fr/LC_MESSAGES/mu.po

```diff
@@ -584,579 +74,633 @@ msgstr ""
+"There was a problem getting the list of files on the device. Please check "
 "Mu's logs for technical information. Alternatively, try unplugging/plugging-"
-"in your micro:bit and/or restarting Mu."
+"in your device and/or restarting Mu."
 msgstr ""
-"Un problème est survenu pendant le listage des fichiers sur la micro:bit. "
+"Un problème est survenu pendant le listage des fichiers sur le périphérique. "
 "Merci de vérifier les logs de Mu pour avoir plus d'informations. Sinon, "
-"essaie de débrancher/rebrancher ta micro:bit, ou bien de redémarrer Mu."
+"essaie de débrancher/rebrancher ton périphérique, ou bien de redémarrer Mu."
 
-#: mu/interface/panes.py:538
+#: mu/interface/panes.py:637
 msgid ""
-"There was a problem copying the file '{}' onto the micro:bit. Please check "
-"Mu's logs for more information."
+"There was a problem copying the file '{}' onto the device. Please check Mu's "
+"logs for more information."
 msgstr ""
 "Un problème est survenu pendant la copie du fichier '{}' sur la micro:bit. "
 "Merci de vérifier les logs de Mu pour avoir plus d'informations."
 
-#: mu/interface/panes.py:546
+#: mu/interface/panes.py:649
 msgid ""
-"There was a problem deleting '{}' from the micro:bit. Please check Mu's logs "
+"There was a problem deleting '{}' from the device. Please check Mu's logs "
 "for more information."
 msgstr ""
-"Un problème est survenu pendant la suppression du fichier '{}' de la micro:"
-"bit. Merci de vérifier les logs de Mu pour avoir plus d'informations."
+"Un problème est survenu pendant la suppression du fichier '{}' sur le "
+"périphérique. Merci de vérifier les logs de Mu pour avoir plus "
+"d'informations."
 
-#: mu/interface/panes.py:554
+#: mu/interface/panes.py:661
 msgid ""
-"There was a problem getting '{}' from the micro:bit. Please check Mu's logs "
-"for more information."
+"There was a problem getting '{}' from the device. Please check Mu's logs for "
+"more information."
 msgstr ""
-"Un problème est survenu pendant la lecture du fichier '{}' depuis la micro:"
-"bit. Merci de vérifier les logs de Mu pour avoir plus d'informations."
+"Un problème est survenu pendant la lecture du fichier '{}' depuis le "
+"périphérique. Merci de vérifier les logs de Mu pour avoir plus "
+"d'informations."
 
-#: mu/logic.py:69
+#: mu/logic.py:94
 msgid "Hello, World!"
 msgstr "Bonjour le monde!"
 
-#: mu/logic.py:70
+#: mu/logic.py:95
 msgid ""
 "This editor is free software written in Python. You can modify it, add "
 "features or fix bugs if you like."
 msgstr ""
 "Cet éditeur est un logiciel libre écrit en Python. Tu peux le modifier, "
 "ajouter des fonctionnalités ou corriger des bogues si tu le souhaites."
 
-#: mu/logic.py:72
+#: mu/logic.py:99
 msgid "This editor is called Mu (you say it 'mew' or 'moo')."
-msgstr "Cet éditeur s'appelle Mu (ca se prononce 'miou' ou 'moo')."
+msgstr "Cet éditeur s'appelle Mu (ça se prononce 'miou' ou 'mou')."
```

Comment:
Does the name from from Greek letter µ @ntoll ?  If so, it would be fine to remove the parenthese, we pronounce both µ and `mu` as `\my\`.